phpstan-baseline.neon

```diff
@@ -0,0 +1,46 @@
+parameters:
```

Comment:
would be cool to get the baseline empty though ;-)

Reply:
Yeah would be great. Some of it from UserMatcher/AccountMatcher are removed now. The others are a nice future improvement.

Reply:
Add types for parameters in ContextXMLRenderer, removed from baseline